raiden/tests/utils/smartcontracts.py

```diff
@@ -56,17 +56,17 @@ def deploy_tokens_and_fund_accounts(
     the raiden registry.
 
     Args:
-        token_amount (int): number of units that will be created per token
-        number_of_tokens (int): number of token instances that will be created
-        deploy_service (BlockChainService): the blockchain connection that will deploy
```

Comment:
Hey hmmm seems this comment is incomplete here. "The blockchain connection that will deploy ... ?"

I think I recall mentioning this in someone else's PR in the past but I guess rebasing reintroduced it? Can you fix it and also adjust the comment for the new awesome name of "ProxyManager"?